.github/workflows/static-analysis.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         TAG:
           [
             "frontend-analytics",
+            "frontend-asset-loader",
```

Comment:
This looks good, I think we need to do the same in the CD pipeline to allow us to select it for deployment?